### PR TITLE
BaSP-TG-33: timesheets unit testing get by id delete edit

### PR DESCRIPTION
**Pull request by GuidoG95**

### CODE REVIEW

File: src/tests/timeSheets.test.js [hunk 1]
```diff
@@ -1 +1,89 @@
-test('Delete this test', () => {});
```
Comment (DinoStampella) [inline src/tests/timeSheets.test.js:1]: You can delete this line

**Review by EmanuelMuises95 — APPROVED** [review]

**Review by EmanuelMuises95 — DISMISSED** [review]

File: src/tests/timeSheets.test.js (hunk 1 above)
Comment (GuidoG95) [reply src/tests/timeSheets.test.js:1]: It's already deleted

File: src/tests/timeSheets.test.js [hunk 2]
```diff
@@ -1 +1,89 @@
...
+import app from '../app';
+import TimeSheets from '../models/TimeSheets';
+import timeSheestsSeed from '../seeds/timeSheets';
+
+beforeAll(async () => {
+  await TimeSheets.collection.insertMany(timeSheestsSeed);
+});
+
+describe('GETBYID /api/timeSheet', () => {
+  test('Status of a valid ID', async () => {
```
Comment (JulianFloresDev) [inline src/tests/timeSheets.test.js:11]: Can it be more specific? I mean something like "Status code should be 200 if sent a valid ID" for example

File: src/tests/timeSheets.test.js [hunk 3]
```diff
@@ -1 +1,89 @@
...
+    const response = await request(app).get('/api/timeSheet/6352e89b4760bc22934f3507').send();
+    expect(response.status).toBe(200);
+  });
+  test('Error of a valid ID', async () => {
```
Comment (JulianFloresDev) [inline src/tests/timeSheets.test.js:15]: Same as above.

I suggest "Error property should be false if sent a valid ID"

File: src/tests/timeSheets.test.js [hunk 4]
```diff
@@ -1 +1,89 @@
...
+    const response = await request(app).get('/api/timeSheet/6352e89b4760bc22934f3507').send();
+    expect(response.error).toBeFalsy();
+  });
+  test('Status of non existent ID', async () => {
```
Comment (JulianFloresDev) [inline src/tests/timeSheets.test.js:19]: Same as above

File: src/tests/timeSheets.test.js [hunk 5]
```diff
@@ -1 +1,89 @@
...
+    const response = await request(app).get('/api/timeSheet/6352e89b4760bc22934f1234').send();
+    expect(response.status).toBe(404);
+  });
+  test('Error of non existent ID', async () => {
```
Comment (JulianFloresDev) [inline src/tests/timeSheets.test.js:23]: Same as above

**Comment by JulianFloresDev** [comment]
I suggest giving a little more description to the tests that are intended to be tested for better suit of testing practices; since the tests should be descriptive of the situation that is intended to be evaluated.

File: src/tests/timeSheets.test.js (hunk 2 above)
Comment (GuidoG95) [reply src/tests/timeSheets.test.js]: changed

File: src/tests/timeSheets.test.js (hunk 4 above)
Comment (GuidoG95) [reply src/tests/timeSheets.test.js]: changed

File: src/tests/timeSheets.test.js [hunk 6]
```diff
@@ -1 +1,89 @@
...
+    const response = await request(app).get('/api/timeSheet/6352e89b4760bc22934f1234').send();
+    expect(response.error).toBeTruthy();
+  });
+  test('Status of invalid ID', async () => {
```
[the remark this answers is not in the document]
Comment (GuidoG95) [reply src/tests/timeSheets.test.js]: changed

File: src/tests/timeSheets.test.js [hunk 7]
```diff
@@ -1 +1,89 @@
...
+    const response = await request(app).get('/api/timeSheet/bfdsiohsb').send();
+    expect(response.status).toBe(400);
+  });
+  test('Error of invalid ID', async () => {
```
[the remark this answers is not in the document]
Comment (GuidoG95) [reply src/tests/timeSheets.test.js]: changed

File: src/tests/timeSheets.test.js (hunk 5 above)
Comment (GuidoG95) [reply src/tests/timeSheets.test.js]: changed

File: src/tests/timeSheets.test.js (hunk 3 above)
Comment (GuidoG95) [reply src/tests/timeSheets.test.js]: changed

**Review by ignacioaristo — APPROVED** [review]

**Review by Licamus — APPROVED** [review]

**Pull request merged**